plugins/integration/providers/voicebase/VoicebaseOptions.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @package plugins.voicebase
+ */
+class VoicebaseOptions
+{
+	/**
+	 * voicebase api-key
+	 */
+	public $apiKey;
+	
+	/**
+	 * Voicebase password
+	 */
+	public $apiPassword;
+	
+	function __construct($params)
```

Comment:
Please accept the two argument instead of associative array, it's much more clear.